server/src/main/java/org/candlepin/controller/PoolManager.java

```diff
@@ -66,14 +66,6 @@
 
     Pool convertToMasterPool(Subscription subscription);
 
-    /**
-     * Updates the pools using the information stored in the given pool. Because
-     * the input subscription is used to lookup pools, the ID field must be set
-     * for this method to operate properly.
-     *
-     * @param pool
-     *        The pool to use for updating the associated pools
-     */
```

Comment:
Why delete the JavaDoc here?  If it's wrong, I'd rather have it corrected than removed entirely.

Reply:
Whoops. Removed it, intended to add it back, got distracted. I'll add that back on.